src/test/java/homeworks/MyLoginTests.java

```diff
@@ -0,0 +1,202 @@
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class MyLoginTests extends MyBaseTest {
+String url= "https://bbb.testpro.io/";
+
+//    @DataProvider
+//    public static Object[][] invalidCredentials() {
+//        return new Object[][]{};
+//    }
+
+
+    @Test( enabled = false, priority = 0,dataProvider = "invalidCredentials")
+    public void LoginEmptyEmailPasswordTest () {
+        Assert.assertEquals(driver.getCurrentUrl(), url);
+    }
+    @DataProvider(name="inavildCredentials")
+    public static Object[][] getCredentials(){
+
+        return new Object[][]{
+                {"invalid@class.com","invalidPass"},
+                {"d@class.com",""},
+                {"",""},
+
+        };
+}
+
+
+
+    @Test (priority = 0)
+    public void LoginValidEmailPasswordTest () {
+        login();//lets try to login again
+        By avatarIcon = By.cssSelector("[class='view-profile']");
+        wait.until(ExpectedConditions.visibilityOfElementLocated(avatarIcon));
+        WebElement avatarIconElement = driver.findElement(avatarIcon);
+        Assert.assertTrue(avatarIconElement.isDisplayed());
+
+
+    }
+    public boolean isAvatarIconPresent() {
+        WebElement avatarIconElement= driver.findElement(By.xpath("//img[contains(@alt,'Avatar of')]"));
+        return avatarIconElement.isDisplayed();
+    }
+
+//    @Test
+//    public void Shuffle(){
+//        AllSongsPage allSongsPage = new AllSongsPage(driver);
+//        LoginPage loginPage = new LoginPage(driver);
+//
+//        HomePage homePage = loginPage.login();
+//
+//        homePage.clickOnAllSongs()
+//                .shuffle();
+//        Assert.assertTrue(homePage.isSongPlaying());
+//
+//    }
+
+//    @Test
+//    public void PlayASongFromAllSongs(){
+//        LoginPage loginPage = new LoginPage(driver);
+//        HomePage homey = new HomePage(driver);
+//        AllSongsPage singysongy = new AllSongsPage(driver);
+//
+//        //Step1: Login
+//        loginPage.login();
+//        //Step2: Click on All Songs
+//        homey.clickOnAllSongs();
+//        //Step3: double click on first song
+//        singysongy.doubleClickFirstSong();
+//        //Step4: check if song is playing
+//        Assert.assertTrue(homey.isSongPlaying());
+//    }
+
+
+//    @Test(enabled = false, priority = 2)
+//    public void LoginInvalidEmailPasswordTest () throws InterruptedException {
+//
+////        provideEmail();
+////        providePassword();
+//
+//        provideEmail("dem@class.com");
+//        providePassword("");
+//
+//        clickSubmitBtn();
+//        By avatarIconLocator = By.xpath("//img[contains(@alt,'Avatar of')]");
+//        WebElement avatarIcon = driver.findElement(avatarIconLocator);
+//        wait.until(ExpectedConditions.elementToBeClickable(avatarIconLocator));
+//        Assert.assertTrue(avatarIcon.isDisplayed());
+//        Assert.assertEquals(driver.getCurrentUrl(), url);
+//
+//    }
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.